registration/validators.py

```diff
@@ -43,8 +43,8 @@ def validate_student(student):
     validate_student_information_role(student.information, student.role)
 
 
-def validate_mc_options(field):
-    if not all(isinstance(option, str) for option in field.options):
+def validate_mc_options(options):
```

Comment:
this was previously erroring out in the django admin (and later would in the endpoints too) because field-level validators only get run on the field's value!

Reply:
gotcha, thank you for the fix :)